server.js

```diff
@@ -9,6 +9,8 @@ var require = require(),
     io = require('socket.io');
 require('dotenv').config();
 
+require('dotenv').config();
```

Comment:
You might want to take a second look at this.
I think we'll be getting duplicates into the repo.